python/tk_motionbuilder/menu_generation.py

```diff
@@ -263,8 +261,8 @@ def __strip_unicode(self, val):
         """
         Get rid of unicode
         """
-        if val.__class__ == unicode:
-            val = unicodedata.normalize("NFKD", val).encode("ascii", "ignore")
+        if type(val) is six.text_type:
```

Comment:
This logic was copy pasted from the Nuke engine. I doubt it's actually necessary. I've seen it in other engines during the port an have removed the code without any noticeable side effect.

There's even a comment at the bottom of the diff that mentions Nuke.

Reply:
It seems Motionbuilder (Py2) can't handle unicode characters regardless in the menu item names. I tested with code before the py3 changes, and after, and both don't work. I've simplified the code as you suggested however.
Motionbuilder 2021 (py3) does work however.